src/database/DataTypes/LocationMovement.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import Realm from 'realm';
+
+export class LocationMovement extends Realm.Object {}
```

Comment:
`ItemBatchLocationMovement`.. `LocationMovement`... i dunno!

Reply:
I vote for `LocationMovement`